rn/Teacher/src/modules/assignments/AssignmentList.js

```diff
@@ -273,6 +273,10 @@ export class AssignmentList extends Component<AssignmentListProps, State> {
       this.data = this.prepareListData()
     }
 
+    if (this.props.screenTitle === i18n('Grades')) {
```

Comment:
Instead of comparing the title, could you send in a prop from `mapStateToProps` that is only true from the grades `mapStateToProps`? Much like you did with `courseCode`? Comparing the title seems a tad bit more brittle to me.

Reply:
Also, a test for this would be good.

Reply:
Yeah, i like this suggestion.